### ISSUE TITLE
Upload stalls even if the peer is connected

### ISSUE THREAD

**Issue by amiyatulu** [beakerbrowser/homebase#20]
Some of the folders are not getting uploaded to homebase achieve directory, even if it shows peers are connected. How can I solve it?

**Comment by pfrazee** [comment]
How are you seeing that the upload is stalled? Are you examining the homebase data directory?

What happens if you restart homebase?

**Comment by pfrazee** [comment]
(Also: Sorry for the issue!)

**Comment by amiyatulu** [comment]
Yes, examining the homebase data directory. Sometimes its getting uploaded, sometimes not. Also many times homebase server peer are not getting connected with computer. 
![screenshot from 2018-09-11 16-09-19](https://user-images.githubusercontent.com/2051004/45355242-85ae6980-b5dd-11e8-8b09-d7006a1cdd9c.png)

**Comment by pfrazee** [comment]
Yeah, looks like a connectivity issue. Sometimes restarting homebase can solve that. I'll keep you updated while we work on this.
